### PR TITLE
docs(setup): increase clarity in instructions

### PR DESCRIPTION
**Pull request by TheMoonWalker1**
Increased clarity of the instructions in files postinstall.rst and vagrant.rst. Also have deleted unnecessary characters in vagrant.rst.

### CODE REVIEW

File: docs/setup/postinstall.rst [hunk 1]
```diff
@@ -11,7 +11,7 @@ After you connect to your Vagrant box, make sure you are in the ``intranet`` dir
 
 You will then need to run ``python manage.py migrate`` to set up the Postgres database.
 
-You can then start the built-in Django web server with ``fab runserver``. Now that you are running the development server, open a browser to http://127.0.0.1:8080 and log in (the default master password is ``swordfish``). If it fails, check the output of ``python manage.py runserver``.
+You can then start the built-in Django web server with ``fab runserver``. Now that you are running the development server, open a browser to http://127.0.0.1:8080 and log in (the default master password is ``swordfish``). Make sure to create an user by doing the steps in the following section in order to login. If it still fails, check the output of ``python manage.py runserver``.
```
Comment (anonymoose2) [inline docs/setup/postinstall.rst]: This was covered in #976. Please remove this set of changes; they will only cause conflicts.

File: docs/setup/vagrant.rst [hunk 2]
```diff
@@ -2,6 +2,8 @@
 Setting up Vagrant
 ******************
 
+**The following commands should be executed in your host computer's console. You may want to create a new directory for easy access in the future.**
```
Comment (anonymoose2) [inline docs/setup/vagrant.rst]:
```suggestion
**The following commands should be executed in your host computer's console.**
```

Cloning the Git repo will already create a new directory. Instructing users to create one is redundant.

File: docs/setup/vagrant.rst [hunk 3]
```diff
@@ -20,7 +22,9 @@ In the ``config`` directory, copy the file ``devconfig.json.sample`` to ``devcon
...
+Run ``vagrant up && vagrant reload`` and wait while the development environment is set up. When you are asked to select a network interface for bridging, enter the number corresponding to one that is active. You may not need to select a network interface for bridging, which would allow you to skip this step. To automatically select this interface in the future, set the "network_interface" key in ``devconfig.json`` to the name of the interface you selected (e.g. ``"en0: Wi-Fi (AirPort)"``). There may be repeated warnings similar to ``Remote connection disconnect`` and ``Warning: connection aborted. Retrying...`` on the second ``vagrant up``. After several minutes they will stop. Once the provisioning process is complete, run ``vagrant ssh`` to log in to the development box.
```
Comment (anonymoose2) [inline docs/setup/vagrant.rst]: This is a little confusing. I would rather break the bridging instructions into a separate paragraph and say "If you are asked to select a network interface for bridging..."

Then the "There may be repeated warnings" section would go in another paragraph.

File: docs/setup/vagrant.rst [hunk 4]
```diff
@@ -20,7 +22,13 @@ In the ``config`` directory, copy the file ``devconfig.json.sample`` to ``devcon
...
+Run ``vagrant up && vagrant reload`` and wait while the development environment is set up. 
+
+If you are asked to select a network interface for bridging, enter the number corresponding to one that is active. To automatically select this interface in the future, set the "network_interface" key in ``devconfig.json`` to the name of the interface you selected (e.g. ``"en0: Wi-Fi (AirPort)"``). 
+
+There may be repeated warnings similar to ``Remote connection disconnect`` and ``Warning: connection aborted. Retrying...`` on the second ``vagrant up``. After several minutes they will stop. Once the provisioning process is complete, run ``vagrant ssh`` to log in to the development box.
```
Comment (theo-o) [inline docs/setup/vagrant.rst]:
```suggestion
There may be repeated warnings similar to ``Remote connection disconnect`` and ``Warning: connection aborted. Retrying...`` on the second ``vagrant up``. After several minutes, they will stop. Once the provisioning process is complete, run ``vagrant ssh`` to log in to the development box.
```

File: docs/setup/vagrant.rst [hunk 5]
```diff
@@ -20,7 +22,13 @@ In the ``config`` directory, copy the file ``devconfig.json.sample`` to ``devcon
...
+
+**The following commands should be executed in the console of your development box.**
```
Comment (theo-o) [inline docs/setup/vagrant.rst]:
```suggestion
**The following commands should be executed in the console of your Vagrant development box.**
```

**Comment by theo-o** [comment]
@TheMoonWalker1 You have not the squashed the commits you are adding in this PR. Take a look at https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git for some guidance.

**Comment by TheMoonWalker1** [comment]
I'm currently working on squashing the commits in the PR.

File: intranet/static/css/dark/eighth.schedule.scss [hunk 6]
```diff
@@ -1,6 +1,6 @@
 @media (min-width: 500px) {
     .schedule-activity-page.floating .eighth-header {
-        background-color: inherit;
+        background-color: black;
```
Comment (anonymoose2) [inline intranet/static/css/dark/eighth.schedule.scss]: This was merged in a PR a while ago. Can you try rebasing against the latest `dev` again?

Comment (TheMoonWalker1) [reply intranet/static/css/dark/eighth.schedule.scss]: Oops, it seems like I merged one too many commits. I will try rebasing once again.

**Comment by TheMoonWalker1** [comment]
I think I got everything up to standards now.

**Comment by anonymoose2** [comment]
> I think I got everything up to standards now.

The commit message is still the same...

**Comment by TheMoonWalker1** [comment]
> > I think I got everything up to standards now.
> 
> The commit message is still the same...

There. I just forgot to push.

**Comment by anonymoose2** [comment]
> > > I think I got everything up to standards now.
> > 
> > 
> > The commit message is still the same...
> 
> There. I just forgot to push.

Um... still the same. Did you force push?

**Comment by TheMoonWalker1** [comment]
> > > > I think I got everything up to standards now.
> > > 
> > > 
> > > The commit message is still the same...
> > 
> > 
> > There. I just forgot to push.
> 
> Um... still the same. Did you force push?

No, Let me try that

**Comment by TheMoonWalker1** [comment]
Ok... Now I think everything is up to standards

**Pull request merged**